### PR TITLE
refactor: Replace react-bootstrap component imports

### PR DESCRIPTION
**Pull request by Djo1e**
Read issue https://github.com/joinmarket-webui/joinmarket-webui/issues/205 for more information.

This PR changes all the components from `react-bootstrap` to be imported as recommended by the [react-bootstrap docs](https://react-bootstrap.github.io/getting-started/introduction#importing-components)

`import * as rb from 'react-bootstrap` => `import <component> from 'react-bootstrap/<component>`

### CODE REVIEW

File: src/components/Alert.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { useState } from 'react'
-import { Alert as BsAlert } from 'react-bootstrap'
+import BsAlert from 'react-bootstrap/Alert'
```
Comment (theborakompanioni) [inline src/components/Alert.jsx:2]: This does not seem quite right. `Alert` is imported as `BsAlert` as our own component is named `Alert`.

File: src/components/Earn.jsx [hunk 2]
```diff
@@ -31,10 +38,10 @@ const YieldgenReport = ({ lines, maxAmountOfRows = 25 }) => {
 
   return (
     <div className="mt-2 mb-3">
-      {empty && <rb.Alert variant="info">{t('earn.alert_empty_report')}</rb.Alert>}
+      {empty && <Alert variant="info">{t('earn.alert_empty_report')}</Alert>}
```
Comment (theborakompanioni) [inline src/components/Earn.jsx:41]: As far as I can see our own `Alert` component does not support `children`, but a `message` property. Does this work on your end?

Comment (Djo1e) [reply src/components/Earn.jsx:41]: Nice catch! This is supposed to be a react-bootstrap component, I accidentally imported the local one. Fixed!

File: src/components/Alert.jsx (hunk 1 above)
Comment (Djo1e) [reply src/components/Alert.jsx:2]: Should I name it differently? Maybe `RBAlert`?

Comment (Djo1e) [reply src/components/Alert.jsx:2]: No, not at all. I'm fine with that. It's also fine if we don't merge this at all :)

Comment (dnlggr) [reply src/components/Alert.jsx:2]: I see you point @theborakompanioni, `rb.*` definitely increases readability. But on the other hand if this is the recommended approach from the library authors, I'd trust that and go with the explicit component imports. We just need to be careful not to confuse our own components with the ones from Bootstrap. No super strong feelings towards either approach though, just a feeling.

Comment (theborakompanioni) [reply src/components/Alert.jsx:2]: Having thought it through again, I am in favor of not merging. And after the thorough analysis of @Djo1e, I see even less reasons to do so. Also, no super strong feelings. Just to let you know.

Comment (dnlggr) [reply src/components/Alert.jsx:2]: Ok then let's not merge. Ok with you @Djo1e?

Comment (Djo1e) [reply src/components/Alert.jsx:2]: Yes, closing this PR

**Pull request closed without merging**